python/fledge/services/core/support.py

```diff
@@ -123,6 +141,15 @@ def add_syslog_storage(self, pyz, file_spec):
             raise RuntimeError("Error in creating {}. Error-{}".format(temp_file, str(ex)))
         pyz.add(temp_file, arcname=basename(temp_file))
 
+    def add_syslog_service(self, pyz, file_spec, service):
+        # The fledge entries from the syslog file
+        temp_file = self._interim_file_path + "/" + "syslog-{}-{}".format(service, file_spec)
+        try:
+            subprocess.call("grep '{}\\[' {} > {}".format(service, _SYSLOG_FILE, temp_file), shell=True)
```

Comment:
This will only work with C Plugins. 
For example: I have 2 north plugins one with C version & another with Python version
```
PI-C[14549]: ERROR: Sending JSON dataType message 'Type' - generic error:
                                         Vs
FogLAMP[23255] INFO: sending_process: sending_process_PI-Python: Started
```

Reply:
The Python PI plugin is obsolete. All north plugin should be being loaded with the C task framework. If we find we have problems with specific ones then I suggest we add them as and when they are an issue. In the meantime we cover the 90% of cases that matter in deployments.

Reply:
> This will only work with C Plugins.

Example shows only with north one. And same apply for Python South plugins as well.

Reply:
We no longer use python south

Reply:
I meant for `south/north python plugins` logs info. 

You mean to collect only South services (which is loaded by C south server) & C-North tasks logs info?